pkg/orchestrator/assetscanprocessor/malware.go

```diff
@@ -33,7 +33,7 @@ func (asp *AssetScanProcessor) getExistingMalwareFindingsForScan(ctx context.Con
 	existingFilter := fmt.Sprintf("findingInfo/objectType eq 'Malware' and foundBy/id eq '%s'", *assetScan.Id)
 	existingFindings, err := asp.client.GetFindings(ctx, models.GetFindingsParams{
 		Filter: &existingFilter,
-		Select: utils.PointerTo("id,findingInfo/malwareName,findingInfo/malwareType,findingInfo/path"),
+		Select: utils.PointerTo("id,findingInfo/malwareName,findingInfo/malwareType,findingInfo/path,findingInfo/ruleName"),
```

Comment:
We don't need this here *unless* GenerateMalwareKey is using it, but that function hasn't been updated.